autogluon/task/tabular_prediction/predictor.py

```diff
@@ -159,9 +159,29 @@ def evaluate_predictions(self, y_true, y_pred, silent=False, auxiliary_metrics=F
         return self._learner.evaluate(y_true=y_true, y_pred=y_pred, silent=silent, 
                                       auxiliary_metrics=auxiliary_metrics, detailed_report=detailed_report)
 
+    def leaderboard(self, dataset=None, silent=False):
+        """
+            Output summary of information about models produced during fit() as a pandas DataFrame.
+            Includes information on test and validation scores for all models, model training times and stack levels.
+
+            Parameters
+            ----------
+            dataset: (TabularDataset object, optional)
+                This Dataset must also contain the label-column with the same column-name as specified during fit().
+                If specified, then the leaderboard returned will contain an additional column 'score_test'
```

Comment:
need to clarify that score_test indicates the predictive performance evaluated on the provided dataset

Reply:
Good point, clarified